lib/gemaker/cmds/base.rb

```diff
@@ -51,6 +52,25 @@ def execute(cmd, error_message = nil)
         error(error_message) if $?.exitstatus != 0
       end
 
+      def execute_gem_bundle
+        execute_within_gem(
+          "bundle config force_ruby_platform true --local",
+          "error setting ruby platform"
+        )
+        execute_within_gem(
+          "bundle install",
+          "error running bundle install"
+        )
+      end
+
+      def execute_within_gem(cmd, error_message = nil)
+        cmd = "(cd #{gem_root_path} && #{cmd})"
+        Bundler.with_unbundled_env do
+          system cmd
+        end
+        error(error_message) if $?.exitstatus != 0
```

Comment:
No me gusta mucho el sobreescribir `cmd`.
Quizás podría ser directo no más:

```suggestion
        Bundler.with_unbundled_env do
          system "(cd #{gem_root_path} && #{cmd})"
        end

        error(error_message) if $?.exitstatus != 0
```